pytorch/training/docker/1.6.0/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,190 @@
+# The tag for the base image is: 11.0-cudnn8-devel-ubuntu16.04
```

Comment:
I don't think this file belongs here? It's for PT 1.6

Reply:
This is kept as a reference. When https://github.com/aws/deep-learning-containers/pull/639 is merged, this file should be deleted from this branch.